packages/vue/src/components/organisms/SfMegaMenu/SfMegaMenu.stories.js

```diff
@@ -2,343 +2,373 @@ import { storiesOf } from "@storybook/vue";
+            },
+          },
+        ],
       };
     },
     template: `<SfMegaMenu 
-        :title="title" 
-        :visible="visible"
+      :visible="true" 
+      title="Man"
+      class="sb-mega-menu"
+    >
+      <SfMegaMenuColumn 
+        v-for="(category, key) in categories" 
+        :key="key" 
+        :title="category.title"
       >
-        <SfMegaMenuColumn 
-          v-for="subcategory in subcategories" 
-          :key="subcategory.header" 
-          :title="subcategory.header"
-        >
-          <SfList>
-            <SfListItem v-for="item in subcategory.items" :key="item.label" >
-              <SfMenuItem
-                :label="item.label" 
-              />
-            </SfListItem>
-          </SfList>
-        </SfMegaMenuColumn>
-      </SfMegaMenu>`,
+        <SfList>
+          <SfListItem v-for="(subcategory, key) in category.subcategories">
+            <SfMenuItem :label="subcategory.title"></SfMenuItem>
+          </SfListItem>
+        </SfList>
+      </SfMegaMenuColumn>
+      <SfMegaMenuColumn title="Featured" class="sf-mega-menu-column--pined-content-on-mobile sf-mega-menu-column--hide-header-on-mobile sb-mega-menu__promo">
+        <div class="sb-mega-menu__banners">
+          <SfBanner
+            v-for="(banner, key) in banners"
+            :key="key"
+            :title="banner.title"
+            :subtitle="banner.subtitle"
+            :image="banner.pictures"
+            class="sb-mega-menu__banner"
+          />
+        </div>
+      </SfMegaMenuColumn>
+    </SfMegaMenu>`,
   }))
-  .add("[slot] aside", () => ({
+  .add("Search results", () => ({
     components: {
       SfMegaMenu,
-      SfMenuItem,
       SfList,
-      AsidePlaceholder,
+      SfBanner,
+      SfProductCard,
+      SfScrollable,
+      SfMenuItem,
     },
-    props: {
-      title: {
-        default: text("title", "Man", "Props"),
-      },
-      asideTitle: {
-        default: text("asideTitle", "Featured", "Props"),
-      },
+    data() {
+      return {
+        categories: [
+          "Long dresses",
+          "Summer dresses",
+          "Dresses",
+          "Coctail dresses",
+          "New dresses",
+          "Dresses on SALE",
+        ],
+      };
+    },
+    template: `<SfMegaMenu
+        :visible="true"
+        title="Search results"
+        class="sb-mega-menu"
+    >
+      <SfMegaMenuColumn title="Categories" class="sf-mega-menu-column--pined-content-on-mobile" style="flex: 0 0 220px; --menu-item-mobile-nav-icon-display: none;">
+        <SfList>
+          <SfListItem v-for="(category, key) in categories">
+            {{category}}
+          </SfListItem>
+        </SfList>
+      </SfMegaMenuColumn>
+      <SfMegaMenuColumn title="Product suggestions" class="sf-mega-menu-column--pined-content-on-mobile" style="flex: 1">
+        <template #title="{title}">
+          <div style="display: flex; align-items: center; --menu-item-mobile-nav-icon-display: none;">
```

Comment:
I think, bottom border should be on 100% width
![image](https://user-images.githubusercontent.com/34887212/82661379-2f0b8680-9c2c-11ea-8b8e-74f9311533ec.png)

Reply:
![Zrzut ekranu 2020-05-22 o 16 00 40](https://user-images.githubusercontent.com/12138170/82675644-65a1cb00-9c45-11ea-81f7-ca6cc15b246e.png)
fixed, thank You 💚